src/resmom/mom_main.c

```diff
@@ -7310,10 +7310,8 @@ main(int argc, char *argv[])
 
 #ifdef RLIMIT_NPROC
 		(void)getrlimit64(RLIMIT_NPROC, &orig_nproc_limit); /* get for later */
-		if (setrlimit64(RLIMIT_NPROC, &rlimit) == -1) {    /* set unlimited */
+		if (setrlimit64(RLIMIT_NPROC, &rlimit) == -1)    /* set unlimited */
 			perror(" setrlimit NPROC");
```

Comment:
convert perror() to log_err() (here and below) so that it goes into mom logs and there's a record of the fact that setting rlimit had failed, which might explain any unexpected behavior as a consequence.